backend/app/testbot/generate_sample_data.py

```
@@ -8,55 +8,52 @@ def generate_data():
+    csv_path_a = os.path.join(chatbot_a_dir, "products.csv")
+    with open(csv_path_a, "w") as f:
+        f.write(csv_data_a.strip())
 
     # Sample data for Chatbot B (XLSX)
-    xlsx_df_B = pd.DataFrame({
+    xlsx_df_b = pd.DataFrame({
         "id": [1, 2],
         "name": ["Widget C", "Gizmo C"],
         "description": ["A lightweight widget", "A powerful gizmo"],
         "price": [16.99, 27.99],
         "category": ["Gadgets", "Gadgets"]
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)